### DISCUSSION TITLE
7.76.1

### DISCUSSION THREAD

**Discussion by bagder** [curl/curl#6890] [General]
[changelog](https://curl.se/changes.html#7_76_1)

<hr /><em>This discussion was created from the release <a href='https://github.com/curl/curl/releases/tag/curl-7_76_1'>7.76.1</a>.</em>

**Comment by sergeevabc** [comment]
Why are the sources released without binaries here, Windows especially?
Every time the notification pops up, I have to dig down the binaries elsewhere.

**Reply by jzakrzewski** [reply]
Because the project only delivers source code. Possible binaries are later build by volunteers. Some of those builds are listed on https://curl.se/download.html
